ModulePresentationsForCAP/gap/ModulePresentationFunctors.gi

```diff
@@ -158,3 +158,60 @@ InstallMethod( FunctorLessGeneratorsRight,
     return functor;
     
 end );
+
```

Comment:
This functor can be computed for any ring with Involution function even if the ring is not commutative. I needed it Left/RightPresentations over exterior algebra.